frontend/tasks/scripts.js

```diff
@@ -61,14 +61,23 @@ gulp.task('scripts-misc', () => {
 });
 
 gulp.task('scripts-generate-static-html', () => {
-  return commonBrowserify('generate-static-html.js', browserify({
+  return browserify({
     entries: [config.SRC_PATH + 'generate-static-html.js'],
     debug: config.IS_DEBUG,
     fullPaths: config.IS_DEBUG,
     transform: [babelify],
     standalone: 'generate-static-html',
     bundleExternal: false
-  }).external(vendorModules));
+  }).external(vendorModules)
```

Comment:
Might be better to tweak `commonBrowserify` to accept an option for output path than copypasta most of it here.